core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2278,6 +2278,37 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_BLOCK_MASTER_CLIENT_POOL_SIZE_MIN =
+      new Builder(Name.USER_BLOCK_MASTER_CLIENT_POOL_SIZE_MIN)
+          .setDefaultValue(0)
+          .setDescription("The minimum number of block master clients cached in the block master "
+              + "client pool. For long running processes, this should be set to zero.")
```

Comment:
what are the implications of setting it to 0, and non-zero?

Reply:
zero means that in case of a leak, we can still recover and close the connection. the other implication is that every 2 mins (default) we might have to open a new connection if we close after inactivity 

non-zero implies that we'll always have at least 1 connection open